properties.schema

```diff
@@ -249,6 +249,84 @@
         }
       },
 
+      "_menu": {
+        "type": "object",
+        "required": false,
+        "title": "Menu",
+        "properties": {
+          "menu-item": {
+            "title": "Menu item colour",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Menu item background colour"
+          },
+          "menu-item-inverted": {
+            "title": "Menu item colour - inverted",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Menu item foreground colour"
+          },
+          "menu-item-border-color": {
+            "title": "Menu item border colour",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Menu item border colour"
+          },
+          "menu-item-progress": {
+            "title": "Menu item progress colour",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Menu item progress fill colour"
+          },
+          "menu-item-progress-inverted": {
+            "title": "Menu item progress colour - inverted",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Menu item progress background colour"
+          },
+          "menu-item-progress-border": {
+            "title": "Menu item progress border colour",
+            "type": "string",
+            "inputType": "ColourPicker",
+            "default": "",
+            "help": "Menu item progres border colour"
```

Comment:
There's a typo in the help text... but really I'm not sure you need it in this and some of the others since it just duplicates the `title` text.